molecule/release-cluster/group_vars/all

```diff
@@ -3,26 +3,18 @@ __pulp_run_once: true
 pulp_configure_firewall: none
 pulp_default_admin_password: password
 pulp_install_plugins:
-  # galaxy-ng: {}
-  # pulp-ansible: {}
-  # pulp-certguard: {}
-  pulp-container: {}
-  # pulp-cookbook: {}
-  # pulp-deb: {}
-  pulp-file: {}
-  # pulp-gem: {}
-  # pulp-maven: {}
-  # pulp-npm: {}
-  # pulp-python: {}
-  # pulp-rpm: {}
+  pulp-file:
+  pulp-rpm:
 pulp_redis_bind: '0.0.0.0:6379'
+pulp_api_bind: "{{ hostvars['centos-7']['ansible_default_ipv4']['address'] }}:24817"
+pulp_content_bind: "{{ hostvars['centos-8']['ansible_default_ipv4']['address'] }}:24816"
```

Comment:
I didn't understand why api/content binds are using different OS vars

Reply:
FYI: I renamed the hosts to:

- pulp-api
- pulp-content
- pulp-database
- pulp-webserver

I'm also only using CentOS 8 and 9, not CentOS 7, until CentOS 7 packages become available.